proseco/catalog.py

```diff
@@ -717,7 +717,7 @@ def merge_cats(fids=None, guides=None, acqs=None, mons=None):
 
     if len(acqs) > 0:
         # TODO: move these into acq.py where possible
-        img_size = get_img_size(len(fids))
+        img_size = guides.img_size or get_img_size(len(fids))
```

Comment:
Still weird that we care here for the display (printing the unused image size of acqs).

Reply:
Well this parameter actually gets to the command loads (i.e. it's in the DOT) and is uplinked to the spacecraft. So technically it is not just printing an unused value. And I'd have to double-check, but I think that it gets through to the ACA search command and might show up in a couple readouts before the OBC sends the guide search command.

Reply:
I thought the acq box readouts were always 8x8 but I was focused more on the PEA side so that's a good question.

Reply:
In obsid 24771 there is an ACQ star with 6x6 in starcheck, and I see IMGSZ11= 1 in the backstop file. So I can't say what is getting sent to the PEA, but I'm pretty sure that 6x6 is in the command loads and is going up to the spacecraft. From that perspective it matters.

Reply:
OK.  Good to have this talk.  The image size for acquisition stars just isn't something I've considered or that we've talked about in this transition... right down to this PR title.

Reply:
Good point about the PR title, fixed.

Reply:
I looked at image telemetry for obsid 8008 and confirmed that the 2-3 readouts of acq-only images were 6x6 before the fids were found. So the size value for ACQ makes it all the way to the PEA.